packages/forms-web-app/src/dynamic-forms/s78-questionnaire/journey.js

```diff
@@ -34,9 +34,14 @@ class S78Journey extends Journey {
 				questions.listedBuildingCheck
 			),
 			new Section('Environmental impact assessment', 'environmental-impact'),
-			new Section('Notifying relevant parties of the application', 'notified').addQuestion(
-				questions.howYouNotifiedPeople
-			),
+			new Section('Notifying people about the application', 'notified')
+				.addQuestion(questions.howYouNotifiedPeople)
+				.addQuestion(questions.uploadSiteNotice)
+				.withCondition(
+					response.answers &&
+						(response.answers[questions.howYouNotifiedPeople.fieldName] == 'site-notice' ||
+							response.answers[questions.howYouNotifiedPeople.fieldName]?.includes('site-notice'))
```

Comment:
As this logic will be added in several places I'm wondering if it could be better to extract this to a function. We could do something like this:


`
.withCondition(checkConditionIncludes(question.howYouNotifiedPeople.fieldName, 'site-notice'))

// still in the constructor
		const checkConditionIncludes = (fieldName, value) => {
			if (!response.answers) return false;
			const field = Array.isArray(response.answers[fieldName]) ? response.answers[fieldName] : [response.answers[fieldName]];
			return field.includes(value);
		}
`